Code/Tools/SceneAPI/SceneCore/Tests/Utilities/PatternMatcherTests.cpp

```diff
@@ -56,6 +68,23 @@ namespace AZ
                 PatternMatcher matcher("^.{4}$", PatternMatcher::MatchApproach::Regex);
                 EXPECT_FALSE(matcher.MatchesPattern("string_to_long_for_regex"));
             }
+
+            TEST(PatternMatcherTest, MatchesPattern_MatchingPrefixInArrayOfPatterns_ReturnsTrue)
+            {
+                AZStd::array<AZStd::string_view, 3> patterns = { "postfix", "xxx", "prefix_" };
```

Comment:
FYI you can make these AZStd::arrays constexpr.
But better than that you can use AZStd::to_array to avoid needing to specify the size of the array
```suggestion
                constexpr auto patterns = AZStd::to_array<AZStd::string_view>({ "postfix", "xxx", "prefix_" });
```